src/main/java/Task.java

```diff
@@ -0,0 +1,12 @@
+public abstract class Task {
+    protected String description;
+    protected boolean isCompleted;
+    Task(String description) throws EmptyArgumentException {
```

Comment:
Remember to use a blank line to separate different parts, constructors, and methods according to Java coding standard.